scipy/stats/_stats_py.py

```diff
@@ -10227,18 +10228,16 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     -----
     Given two probability mass functions, :math:`u`
     and :math:`v`, the first Wasserstein distance between the distributions
-    is:
+    using the Euclidean norm is:
 
     .. math::
 
-        l_1 (u, v) = \inf_{\pi \in \Gamma (u, v)} \int_{\mathbb{R} \times
-        \mathbb{R}} |x-y| \mathrm{d} \pi (x, y)
```

Comment:
The rest looks good. I just think it's better to mention the underlying space associated with the distributions instead of the distance. 

I would suggest updating the text to something like 'Given two probability mass functions on R^n' or '... mass functions on the Euclidean space' without the 'using euclidean norm' part. The updated formula already explains the used distance metric.

Reply:
I agree that the notation makes the used distance metric more clear, but it still needs to be defined. I don't think that all users would be familiar with norms. 

$R^n$ is not necessarily endowed with the Euclidean norm. For instance, you could also use the absolute value as your distance function. I also find you second suggestion to be a bit confusing for two reasons:

1. I have never seen probability mass functions being defined on the Euclidean space.
2. By defining the Wasserstein metric, we place the probability measures on a Wasserstein space.

I think that my suggested edit is more clear. [Wikipedia](https://en.wikipedia.org/wiki/Wasserstein_metric#Deterministic_distributions) uses a similar language:

>and we use the usual [Euclidean norm](https://en.wikipedia.org/wiki/Euclidean_norm) on R n \mathbb {R} ^{n} as the distance function